composer.json

```diff
@@ -399,6 +403,10 @@
 
         "wordpress-48-test": "@composer test -- tests/Integrations/WordPress/V4_8",
 
+        "yii-2-update": "@composer --working-dir=tests/Frameworks/Yii/Version_2_0_26 update",
```

Comment:
On the folder name, is this guaranteed to be version `2.0.26` after running composer update? We only lock the other frameworks to the minor version so that BC's in patch versions can be detected (which has happened a few times). :)

Reply:
On it!

Reply:
According to this, it's like Yii 2.0.14+:

> https://github.com/yiisoft/yii2/blob/master/ROADMAP.md